src/components/RecentSnippets.jsx

```diff
@@ -1,26 +1,34 @@
-import React, { Fragment } from 'react'
+import React, { Fragment, useEffect } from 'react'
 import { connect } from 'react-redux'
 
 import RecentSnippetItem from './RecentSnippetItem'
+
 import { fetchRecentSnippets } from '../actions'
-import { scrollTop } from '../misc/dom'
 
 import '../styles/RecentSnippets.styl'
 
-class RecentSnippets extends React.Component {
-  componentDidMount() {
-    const { dispatch, recent, pagination } = this.props
+const scrollTop = () => {
+  window.scroll({ top: 0, behavior: 'smooth' })
+}
+
+const RecentSnippets = props => {
```

Comment:
probably it would be better to have destructuring here `({ dispatch, pagination, snippets, recent }) => {}`